enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -2253,6 +2253,12 @@ class AdjointGenerator
           }
         }
       }
+      if (looseTypeAnalysis) {
```

Comment:
maybe add forward version as well .

Reply:
Just checked, we don't even support that instruction in Fwd mode.
So I'd leave that for when I actually use it since it probably deserves a new testcase too.